proxy/coco/tests/gossip.rs

```diff
@@ -0,0 +1,189 @@
+use std::time::Duration;
+
+use futures::future;
+use futures::StreamExt as _;
+use tokio::time::timeout;
+
+use librad::net::protocol::ProtocolEvent;
+
+use coco::seed::Seed;
+use coco::{Hash, Urn};
+
+mod common;
+use common::{
+    build_peer, build_peer_with_seeds, init_logging, radicle_project, shia_le_pathbuf,
+    wait_connected,
+};
+
+#[tokio::test]
+async fn announce_solo() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Again, a test I'd have no idea what it's doing without reading dozens of lines of code. This should be ignored lightly, it creates unnecessary strain in the reader that is not as familiar with the whole domain.

Reply:
What would you propose as an alternative?

Reply:
`can_announce_new_project` maybe?